mod/wet4/views/default/event_calendar/show_events.php

```diff
@@ -72,11 +72,10 @@
 	}else{
 		$new_link = elgg_view('output/url', array(
         'href' => "event_calendar/add",
-        'text' => elgg_echo('event_calendar:new1'),
```

Comment:
No, I look in all the site and event_calendar:new1 do not exist. 
Also, we never talk about changing the wording "add event" for something else.